config/authorization/config.ex

```diff
@@ -164,7 +164,7 @@ defmodule Acl.UserGroups.Config do
       "http://publications.europa.eu/ontology/euvoc#Language",
       "http://www.w3.org/2004/02/skos/core#Concept",
       "http://www.w3.org/2004/02/skos/core#ConceptScheme",
-      "http://mu.semte.ch/vocabularies/ext/publicatie/PublicationMetricsExportType",
+      "http://mu.semte.ch/vocabularies/ext/publicatie/Publicatierapporttype",
```

Comment:
I would prefer this to have a name that's more consistent with the others in terms of capitalization.
So http://mu.semte.ch/vocabularies/ext/publicatie/PublicatieRapportType

(zie bijvoorbeeld `case-type -> ext:DossierTypeCode`: https://github.com/kanselarij-vlaanderen/app-kaleidos/blob/development/config/resources/dossier-domain.lisp#L26